### ISSUE TITLE
Scala 3

### ISSUE THREAD

**Issue by shishkin** [exoego/aws-lambda-scalajs-facade#321]
Hi! Any plans to provide Scala 3 artifacts as well?

**Comment by shishkin** [comment]
Actually, just learned how to use Scala 2.13 libs in Scala 3 projects:

```scala
libraryDependencies += ("net.exoego" %%% "aws-lambda-scalajs-facade" % "0.11.0").cross(CrossVersion.for3Use2_13)
```

**Comment by exoego** [comment]
Thanks.
0.12.1 released for Scala 2.13 and 3.x today 🙇
